<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveStrategyDialog</class>
 <widget class="QDialog" name="SaveStrategyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>XML</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/xml.png</normaloff>:/images/xml.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Binary</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/binary.png</normaloff>:/images/binary.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Изображение</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/image.png</normaloff>:/images/image.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
       <item>
        <widget class="QTextEdit" name="teDescription">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbChoose">
       <property name="text">
        <string>Выбрать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
